{{cookiecutter.repo_name}}/.github/workflows/CI.yaml

```diff
@@ -0,0 +1,45 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: {{ '${{ matrix.os }}' }}
+    strategy:
+      max-parallel: 4
+      matrix:
+        os: [macOS-latest, ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Additional info about the build
+      run: |
+        uname -a
+        df -h
+        ulimit -a
+    - name: Create environment for package
+      run: |
+        eval "$(${CONDA}/conda shell.bash hook)" && conda activate
+        python devtools/scripts/create_conda_env.py -n=test -p={{ '${{ matrix.python-version }}' }} devtools/conda-envs/test_env.yaml
+    - name: Install package
+      run: |
+        eval "$(${CONDA}/conda shell.bash hook)" && conda activate test
```

Comment:
Too bad we cannot outrun this. Can we do something like:

```bash
##vso[task.setvariable variable=CONDA_ENV]"$(${CONDA}/conda shell.bash hook) && conda activate test"
eval $CONDA_ENV
```

so that it will be easier to add additions steps?

Reply:
We would still need to `eval $CONDA_ENV` and `conda activate test` in all steps, right? I'd rather be explicit about and not use this Azure magics (where can I find docs about them, btw?) if possible.

Reply:
Found the equivalent docs for GHA: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/development-tools-for-github-actions#set-an-environment-variable-set-env

I am using that to define `CONDA_EXE` across different platforms. We still need to initialize/eval the shell and activate the environments, though.